apps/frontend/src/features/battle/components/setup/ParticipantsList.tsx

```
@@ -14,19 +14,40 @@ export const ParticipantsList = ({ participants, currentParticipantId }: Partici
   const theme = useTheme();
 
   return (
-    <div css={containerStyle}>
-      <h2 css={titleStyle(theme)}>PARTICIPANTS</h2>
+    <div css={containerStyle} role="region" aria-label="참가자 목록">
+      <h2 css={titleStyle(theme)} id="participants-heading">
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)